src/object/srv_obj.c

```diff
@@ -1356,6 +1356,7 @@ ds_iter_vos(crt_rpc_t *rpc, struct vos_iter_anchors *anchors,
 		param.ip_epr.epr_lo = 0;
 		recursive = true;
 		enum_arg->chk_key2big = true;
+		param.ip_flags = VOS_IT_RECX_VISIBLE;
```

Comment:
The enumeration API/RPC should be changed to specify a epoch range (like, [snapshot1 + 1, snapshot2)), right?

Reply:
yes, it should, but it needs VOS support that as I understand.

Reply:
Sure, VOS supports this.

Reply:
ah, good.